packages/flutter/lib/src/material/dropdown.dart

```diff
@@ -1288,6 +1293,10 @@ class DropdownButton<T> extends StatefulWidget {
   /// Defines the corner radii of the menu's rounded rectangle shape.
   final BorderRadius? borderRadius;
 
+  /// Determines whether tapping outside the dropdown will close it.
+  /// Defaults to `true` and cannot be `null`.
```

Comment:
Nonnullable is implicated by the type.

```suggestion
  /// Determines whether tapping outside the dropdown will close it.
  /// Defaults to `true`.
```

Reply:
Pushed the change